src/js/intlTelInput.js.ejs

```diff
@@ -429,7 +429,7 @@ Plugin.prototype = {
     var that = this;
 
     // check for cookie
-    var cookieAutoCountry = ($.cookie) ? $.cookie("itiAutoCountry") : "";
+    var cookieAutoCountry = (!!window.Cookies) ? Cookies.get("itiAutoCountry") : "";
```

Comment:
Its the same difference as in `==` vs `===` (or `!=` vs `!==`).

Without these, what you're comparing is truthy/falsey which works fine for cases like this, but readability-wise, I think this conveys your intent better.

Again, JS is a highly opinionated language and you're the author and this isn't a battle worth fighting; so if you want to change it, please go ahead! :)